plugins/woocommerce/i18n/states.php

```diff
@@ -831,17 +829,19 @@
 		'TN' => __( 'Tamil Nadu', 'woocommerce' ),
 		'TS' => __( 'Telangana', 'woocommerce' ),
 		'TR' => __( 'Tripura', 'woocommerce' ),
-		'UK' => __( 'Uttarakhand', 'woocommerce' ),
 		'UP' => __( 'Uttar Pradesh', 'woocommerce' ),
+		'UK' => __( 'Uttarakhand', 'woocommerce' ),
```

Comment:
Looking at [the source](https://github.com/unicode-org/cldr/blob/2ef784e3a4168bc2a43cd1b5b9839b6636f5899c/common/subdivisions/en.xml#L2362), it looks like this should be:

```suggestion
		'UT' => __( 'Uttarakhand', 'woocommerce' ),
		'UK' => __( 'Uttarākhand', 'woocommerce' ),
```